examples/agents/crawl4ai-agent.mdx

```diff
@@ -0,0 +1,32 @@
+---
+title: Crawl4ai Agent
+---
+
+Create a file `crawl4ai_agent.py` with the following code:
```

Comment:
Lets explain what crawl4ai is or provide a link to their site. We should imagine non-tech or non-ai folks also reading these docs and trying something out.